core/src/com/mygdx/crazybomber/Model/player/Player.java

```diff
@@ -0,0 +1,88 @@
+package com.mygdx.crazybomber.model.player;
+
+public class Player {
+    private boolean _isKnockedUp;
+    private boolean _isAlive;
+    private int _numBombs;
+    private int _speed;
+    private boolean onItem;
+    private int _xCoordinate;
```

Comment:
Should be double, different from bomb cause the player doesnt have to be in the middle of the square all the time.

Reply:
completed